src/vs/workbench/contrib/terminalContrib/suggest/test/browser/terminalSuggestAddon.integrationTest.ts

```diff
@@ -122,6 +150,7 @@ suite('Terminal Contrib Suggest Recordings', () => {
 		test(testCase.name, async () => {
 			const suggestDataEvents: string[] = [];
 			store.add(suggestAddon.onAcceptedCompletion(e => suggestDataEvents.push(e)));
+			store.add(pwshCompletionProvider.onAcceptedCompletion(e => suggestDataEvents.push(e)));
```

Comment:
I think we want the pwsh completion provider to forward this request to the main suggest addon

Reply:
tried that and it caused a test to fail 🫠

Reply:
just one test, so introduces timing issue I think

Reply:
thinking more about it, the `pwshCompletionProvider` is unique - others will not have a similar event. should this be renamed?

Reply:
IE `onRequestCompletions`